onedal/common/hyperparameters.py

```diff
@@ -56,9 +60,12 @@ def __getattribute__(self, __name):
                 return super().__getattribute__(__name)
             elif __name in self.getters.keys():
                 return self.getters[__name]()
-            else:
-                raise ValueError(
-                    f"Unknown '{__name}' name in "
+            try:
+                # return all variables that are in super() class normally
+                return super().__getattribute__(__name)
```

Comment:
Not clear why is it needed for class without any parent. Comment with justification would be great

Reply:
It is still a Python object, so there are builtin attributes, like `__class__`, __sizeof__`, etc. 
They are used in debuggers and making them inaccessible just leads to confusion. I will update the comment to better explain it.